packages/lib/modules/fee-managers/getFeeManagersMetadata.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  fetchAndMapMetadata,
+  lowerCaseAddresses,
+} from '@repo/lib/shared/utils/fetchAndLowercaseAddresses'
+
+const FEE_MANAGERS_METADATA_URL =
+  'https://raw.githubusercontent.com/balancer/metadata/refs/heads/main/fee_managers/index.json'
+
+export type FeeManagersMetadata = {
+  id: string
+  name: string
+  description: string
+  addresses: Record<string, string[]> // chainId -> addresses[]
```

Comment:
Wouldn't a better type for this be something like:
```
`0x${string}`
```
we can even make an alias to make it clearer.